packages/react-icons/build-verify.test.js

```diff
@@ -1233,35 +1176,35 @@ describe('Build Verification', () => {
+          export const AccessibilityCheckmarkRegular = ( /*#__PURE__*/createFluentFontIcon("AccessibilityCheckmarkRegular", "", 2, undefined));
+          export const AccessibilityErrorFilled = ( /*#__PURE__*/createFluentFontIcon("AccessibilityErrorFilled", "", 2, undefined));
+          export const AccessibilityErrorRegular = ( /*#__PURE__*/createFluentFontIcon("AccessibilityErrorRegular", "", 2, undefined));
+          export const AccessibilityMoreFilled = ( /*#__PURE__*/createFluentFontIcon("AccessibilityMoreFilled", "", 2, undefined));
+          export const AccessibilityMoreRegular = ( /*#__PURE__*/createFluentFontIcon("AccessibilityMoreRegular", "", 2, undefined));
+          export const AccessibilityQuestionMarkFilled = ( /*#__PURE__*/createFluentFontIcon("AccessibilityQuestionMarkFilled", "", 2, undefined));
+          export const AccessibilityQuestionMarkRegular = ( /*#__PURE__*/createFluentFontIcon("AccessibilityQuestionMarkRegular", "", 2, undefined));
+          export const AddFilled = ( /*#__PURE__*/createFluentFontIcon("AddFilled", "", 2, undefined));
+          export const AddRegular = ( /*#__PURE__*/createFluentFontIcon("AddRegular", "", 2, undefined));
+          export const AddCircleFilled = ( /*#__PURE__*/createFluentFontIcon("AddCircleFilled", "", 2, undefined));
+          export const AddCircleRegular = ( /*#__PURE__*/createFluentFontIcon("AddCircleRegular", "", 2, undefined));
+          export const AddSquareFilled = ( /*#__PURE__*/createFluentFontIcon("AddSquareFilled", "", 2, undefined));
+          export const AddSquareRegular = ( /*#__PURE__*/createFluentFontIcon("AddSquareRegular", "", 2, undefined));
+          export const AddSquareMultipleFilled = ( /*#__PURE__*/createFluentFontIcon("AddSquareMultipleFilled", "", 2, undefined));
+          export const AddSquareMultipleRegular = ( /*#__PURE__*/createFluentFontIcon("AddSquareMultipleRegular", "", 2, undefined));
+          export const AddStarburstFilled = ( /*#__PURE__*/createFluentFontIcon("AddStarburstFilled", "", 2, undefined));
+          export const AddStarburstRegular = ( /*#__PURE__*/createFluentFontIcon("AddStarburstRegular", "", 2, undefined));
+          export const AddSubtractCircleFilled = ( /*#__PURE__*/createFluentFontIcon("AddSubtractCircleFilled", "", 2, undefined));
+          export const AddSubtractCircleRegular = ( /*#__PURE__*/createFluentFontIcon("AddSubtractCircleRegular", "", 2, undefined));
+          export const AgentsFilled = ( /*#__PURE__*/createFluentFontIcon("AgentsFilled", "", 2, undefined));
+          export const AgentsRegular = ( /*#__PURE__*/createFluentFontIcon("AgentsRegular", "", 2, undefined));
+          export const AgentsAddFilled = ( /*#__PURE__*/createFluentFontIcon("AgentsAddFilled", "", 2, undefined));
+          export const AgentsAddRegular = ( /*#__PURE__*/createFluentFontIcon("AgentsAddRegular", "", 2, undefined));
```

Comment:
there are more icons components added because babel no longer runs on tsc transpiled code - thus the emitted code is not formatted - creating more space in the trimmed snapshot